flang/lib/Parser/unparse.cpp

```diff
@@ -2090,6 +2090,18 @@ class UnparseVisitor {
         ":");
     Walk(std::get<OmpObjectList>(x.t));
   }
+  void Unparse(const OmpInitClause::InteropTypes &x) {
+    Walk(x.v,",");
```

Comment:
```suggestion
    Walk(x.v, ",");
```

Reply:
Addressed the review comment on [#a36032d](https://github.com/swatheesh-mcw/llvm-project/pull/1/commits/a36032d4920bdb75940a3f29597c195893396b0c).